pom.xml

```diff
@@ -48,7 +48,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-
+        <!-- FIXME: powsybl-network-store modules'version is overloaded in the dependencies section.The overloads and this property below have to be removed at next powsybl-ws-dependencies.version upgrade -->
```

Comment:
You can remove all the FIXME as powsybl-ws-dependencies 2.11.0 has been released. You will see it when you will update your branch with main